### PR TITLE
Added RolloutOperatorNotReconciling alert

### PR DESCRIPTION
**Pull request by pracucci**
#### What this PR does
When I upstreamed the multi-zone support to jsonnet (https://github.com/grafana/mimir/pull/1352) I forgot to upstream the `RolloutOperatorNotReconciling` alert we're using at Grafana Labs. Doing it in this PR.

#### Which issue(s) this PR fixes or relates to

N/A

#### Checklist

- [ ] Tests updated
- [ ] Documentation added
- [x] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

File: docs/sources/operators-guide/mimir-runbooks/_index.md [hunk 1]
```diff
@@ -1038,6 +1038,23 @@ How to **investigate**:
   1. The alert fired because of a bug in Mimir: fix it.
   1. The alert fired because of a bug or edge case in the continuous test tool, causing a false positive: fix it.
 
+### RolloutOperatorNotReconciling
+
+This alert fires if the [`rollout-operator`](https://github.com/grafana/rollout-operator) is not successfully reconciling in a namespace.
+
+How it **works**:
+
+- The operator coordinates the rollout of pods between different StatefulSets within a specific namespace and is used to manage multi-zone deployments
```
Comment (treid314) [inline docs/sources/operators-guide/mimir-runbooks/_index.md:1047]: I think saying "Rollout Operator" instead of operator would be more clear.

File: docs/sources/operators-guide/mimir-runbooks/_index.md [hunk 2]
```diff
@@ -1038,6 +1038,23 @@ How to **investigate**:
...
+- The operator is deployed in Mimir cells where some services (e.g. ingesters) are deployed in multi-zone
+- The operator reconciles as soon as there's any change in observed Kubernetes resources or every 5m at most
+
+How to **investigate**:
+
+- Check rollout-operator logs to find more details about the error, e.g. with the following Grafana Loki query:
```
Comment (treid314) [inline docs/sources/operators-guide/mimir-runbooks/_index.md:1053]: Is there anything specific we should be looking for?

**Review by treid314 — APPROVED** [review]
LGTM, minor comments on the runbook

File: docs/sources/operators-guide/mimir-runbooks/_index.md (hunk 1 above)
Comment (pracucci) [reply docs/sources/operators-guide/mimir-runbooks/_index.md:1047]: Done

File: docs/sources/operators-guide/mimir-runbooks/_index.md (hunk 2 above)
Comment (pracucci) [reply docs/sources/operators-guide/mimir-runbooks/_index.md:1053]: In normal conditions, no error is expected to be logged by the rollout-operator. If anything goes wrong, the logged error should have enough details to better understand what's going on.
